x-pack/plugins/enterprise_search/public/applications/shared/indexing_status/indexing_status_errors.tsx

```diff
@@ -24,8 +24,11 @@ export const IndexingStatusErrors: React.FC<IIndexingStatusErrorsProps> = ({ vie
     data-test-subj="IndexingStatusErrors"
   >
     <p>{INDEXING_STATUS_HAS_ERRORS_TITLE}</p>
-    <EuiButton color="danger" fill={true} size="s" data-test-subj="ViewErrorsButton">
-      <EuiLinkTo to={viewLinkPath}>{INDEXING_STATUS_HAS_ERRORS_BUTTON}</EuiLinkTo>
-    </EuiButton>
+
+    <EuiLinkTo to={viewLinkPath}>
+      <EuiButton color="danger" fill size="s" data-test-subj="ViewErrorsButton">
+        {INDEXING_STATUS_HAS_ERRORS_BUTTON}
+      </EuiButton>
+    </EuiLinkTo>
```

Comment:
Actually you can just shortcut this and use `EuiButtonTo`!! 🎉 

```suggestion
    <EuiButtonTo to={viewLinkPath} color="danger" fill={true} size="s" data-test-subj="ViewErrorsButton">
        {INDEXING_STATUS_HAS_ERRORS_BUTTON}
    </EuiButtonTo>
```

(not quite sure if the char width there is correct, you might have to make this change in-IDE so Prettier doesn't complain)